immutablecollections/_immutableset.py

```diff
@@ -286,6 +310,24 @@ def __rsub__(self, other: AbstractSet[V]) -> "AbstractSet[V]":
         """
         return set(item for item in other if item not in self)
 
+    def symmetric_difference(self, other: Iterable[V]) -> "AbstractSet[Union[T, V]]":
+        """
+        Return the union of elements that are in this set but not the other and elements that are
+        in the other set but not this set (`(self - other) | (other - self)`).
+        """
+        if isinstance(other, AbstractSet):
+            return self ^ other
+        return self ^ immutableset(other)
+
+    def copy(self) -> "ImmutableSet[T]":
+        """
+        For efficiency reasons, rather than provide a copy of this set, return this set directly.
```

Comment:
@berquist : I would reword this even more strongly - something like:

```
Return this set.

Because ImmutableSet is immutable, there is never a reason to copy it. This method is provided for compatibility with `set` and `frozenset`
```

Reply:
Looks like `frozenset` does the same thing, only `set.copy()` makes a new object.